.changeset/real-rockets-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-react": patch
+---
+
+Alert: Prepare for i18n support
```

Comment:
These changes should not affect consumers. Do we still want a changeset, or should wait until we make the translations and the provider available?